src/Sylius/Bundle/ThemeBundle/Command/AssetsInstallCommand.php

```diff
@@ -65,7 +65,13 @@ protected function execute(InputInterface $input, OutputInterface $output): void
             $symlinkMask = max($symlinkMask, AssetsInstallerInterface::RELATIVE_SYMLINK);
         }
 
-        $assetsInstaller->installAssets($input->getArgument('target'), $symlinkMask);
+        $targetDir = $this->getContainer()->getParameter('sylius_core.public_dir');
+
+        if ($input->getArgument('target')) {
```

Comment:
`getArgument` -> `hasArgument`

Reply:
I'm creating a PR to revert it, since it's introduced new bug and i've missed it in my tests